modules/ROOT/pages/_partials/rn-known-issues-ref.adoc

```diff
@@ -1,7 +1,7 @@
 [id='rn-known-issues-ref']
 = Known issues
 
-This section lists known issues with  {PRODUCT} {PATCH_VERSION_NO}:
+This section lists known issues with  {PRODUCT} {MINOR_VERSION_NO}:
 
 * In the _Managing and protecting APIs using API management_ Solution Pattern, users must have administrator privileges to create a product in 3scale. Administrator privileges are not required for other Solution Patterns.
```

Comment:
May be something to address in 1.8 when we have more time, but ... 

I'm confused - do they need 3scale admin privs, or to be an RHMI admin who belongs to the dedicated-admins group?  If the former, the workaround would be to ask an RHMI admin who belongs to the dedicated-admins group to go into 3scale and give the user 3scale admin privs, I think.

Reply:
@jciroli-rh I'm cheating on the attributes because I'm on the 1.7 Branch and I was too lazy to add Yet Another Attribute to keep the
{PATCH_VERSION_NO} straight for both 1.7.0 and 1.7.1. (Sorry, prolly not 'splaining that very well.)

Reply:
About the 3scale Sol Pattern - I have no idea, especially since this is for our lesser known 1.x line on OSD3? Trusting Brian's text and all of the previous reviews for 1.7.0 on this one for now.